src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/BatchingSqlJournal.cs

```diff
@@ -800,8 +801,15 @@ private void FailChunkExecution(ChunkExecutionFailure message)
                         replayAll.ReplyTo.Tell(new EventReplayFailure(cause));
                         break;
 
+                    case SelectCurrentPersistenceIds select:
+                        // SqlJournal handled this failure case by using the default PipeTo failure
+                        // handler which sends a Status.Failure message back to the sender.
+                        select.ReplyTo.Tell(new Status.Failure(cause));
+                        break;
+
                     default:
-                        throw new Exception($"Unknown persistence journal request type [{request.GetType()}]");
+                        Log.Error(cause, $"Batching failure not reported to original sender. Unknown batched persistence journal request type [{request.GetType()}].");
```

Comment:
Throwing would swallow the original failure cause, create confusing error message, and blow up BatchingSqlJournal.